mri/operators/fourier/utils.py

```diff
@@ -143,8 +143,8 @@ def get_stacks_fourier(kspace_loc, volume_shape):
     # Find the mask used to sample stacks in z direction
     full_stack_z_loc = convert_mask_to_locations(
         np.ones(volume_shape[2]),
-    )[:, 0]
-    sampled_stack_z_loc = np.unique(kspace_loc[:, 2])
+    )[:, 0].astype('float16')
+    sampled_stack_z_loc = np.unique(kspace_loc[:, 2]).astype('float16')
 
     try:
         # We use np.isclose rather than '==' as the actual z_loc comes
```

Comment:
As the change to np.isclose is already done, maybe we can remove this comment @mathrip ? As it is a small change, maybe we can do it in this PR itself.

Reply:
Yes, I agree